### PR TITLE
updating image naming conventions

### PR DESCRIPTION
**Pull request by srelf-ukcloud**
Please review but do not release until the OpenStack team are ready :D

### CODE REVIEW

File: articles/openstack/ostack-ref-public-image-versioning.md [hunk 1]
```diff
@@ -19,30 +19,24 @@ toc_mdlink: ostack-ref-public-image-versioning.md
...
-To provide you with both predictability and access to the latest updates, without having to provide an unmanageably large catalog, we use a long-term support (LTS) approach for our OpenStack public image catalog.
-
 ## Image versioning
 
-In our public catalog, we keep the original version of the image in the catalog for the lifetime of that distribution (providing predictability). We also maintain a second version of the release of the template that we patch and replace as necessary during the distribution (providing you with access to the latest version).
-
-This means that:
-
-- The LTS (original version) is predictable both in deployment state and UUID within OpenStack
+We will be providing updated images once a quarter for our supported images. We will not be removing images from the platform until they are no longer in use by any customers. We will be adopting a naming convention that will allow you to easily identify images.
```
Comment (suehighmoor) [inline articles/openstack/ostack-ref-public-image-versioning.md]: I'd suggest changing the tense of this to be a bit simpler:

We update our supported images once a quarter. We will not remove images from the platform until they are no longer in use by any customer.

I'd also suggest moving the bit about naming conventions to the next section.

File: articles/openstack/ostack-ref-public-image-versioning.md [hunk 2]
```diff
@@ -19,30 +19,24 @@ toc_mdlink: ostack-ref-public-image-versioning.md
...
 
-- The current version is not predictable in deployment as patches are applied and the UUID changes when the image is updated
+UKCloud always recomends using the latest availible image, and recomends that all instances have their packages updated as often as possible.
 
 ## Image naming convention
 
 Within the public catalog, images are named using the following naming convention:
```
Comment (suehighmoor) [inline articles/openstack/ostack-ref-public-image-versioning.md]: Within the public catalog, we use the following image naming convention to enable you to easily identify images and select the best image to meet your requirements:

File: articles/openstack/ostack-ref-public-image-versioning.md [hunk 3]
```diff
@@ -19,30 +19,24 @@ toc_mdlink: ostack-ref-public-image-versioning.md
...
 
-`<distributionName>-<distributionVersionNumber>-LTS`
+`<distributionName>-<distributionVersionNumber>-CURRENT` - This will be the most recent image for the quarter.
 
-`<distributionName>-<distributionVersionNumber>-CURRENT`
+`<distributionName>-<distributionVersionNumber>-DATE` - The data indicates when the image was rotated from being the current image.
```
Comment (suehighmoor) [inline articles/openstack/ostack-ref-public-image-versioning.md]: Typo: data => date

Older image, where the date indicates when the image was rotated out from being the current image.

Should <DATE> be in angled brackets to indicate that it is a variable?

**Comment by suehighmoor** [comment]
@srelf-ukcloud, let me know when you're ready to go with this.

**Review by nebupm — APPROVED** [review]

**Pull request merged**